packages/style-engine/class-wp-style-engine-css-rule.php

```diff
@@ -92,17 +91,26 @@ public function add_declarations( $declarations ) {
 		}
 
 		/**
-		 * Sets the at_rule.
+		 * Sets the rules group.
 		 *
-		 * @param string $at_rule A CSS nested @rule, such as `@media (min-width: 80rem)` or `@layer module`.
+		 * @param string $rules_group A parent CSS selector in the case of nested CSS, or a CSS nested @rule, such as `@media (min-width: 80rem)` or `@layer module`.
 		 *
 		 * @return WP_Style_Engine_CSS_Rule Returns the object to allow chaining of methods.
 		 */
-		public function set_at_rule( $at_rule ) {
-			$this->at_rule = $at_rule;
+		public function set_rules_group( $rules_group ) {
+			$this->rules_group = $rules_group;
 			return $this;
 		}
 
+		/**
+		 * Gets the rules group.
+		 *
+		 * @return string
```

Comment:
Just noticed that `null` is a valid return type, so should we update this to reflect that?

```suggestion
		 * @return string|null
```